Contributing/RELEASING.md

```diff
@@ -1,6 +1,6 @@
 #### Releasing:
 1. Create a `fastlane/.env` file with your GitHub API token (see `fastlane/.env.SAMPLE`). This will be used to create the PR, so you should use your own token so the PR gets assigned to you. 
-2. Run `bundle exec fastlane bump`
+2. Run `bundle exec fastlane ios bump_ios`
```

Comment:
I couldn't call the common `bump` lane directly, since they would share the name and end up in an inifinite loop. I thought about a couple options, like calling the terminal fastlane command from the fastfile with `sh` or moving the logic to a different method that can be shared by the common `bump` lane and the ios `bump` lane, but I thought those weren't clean. In the end, I renamed the common `bump` lane to `bump_version_update_changelog_create_pr`. With this, each platform can create their own `bump` lane that is just a wrapper around the `bump_version_update_changelog_create_pr` lane, which I think is fine and if we have to add platform specific logic in the future is probably cleaner.